### ISSUE TITLE
Different backend per game settings

### ISSUE THREAD

**Issue by kathyrollo** [hrydgard/ppsspp#16211]
### What should happen

The backend renderer (OGL, D3D9, D3D11, Vulkan) should be set at both global and per game settings. Currently, changing the per game settings also modifies the global config.

### Who would this benefit

All users. Some games perform better on different backends. I use Vulkan for global but GoW series tend to perform better on D3D11.

### Platform (if relevant)

Windows

### Games this would be useful in

God of War: Chains of Olympus, God of War: Ghost of Sparta

### Other emulators or software with a similar feature

DuckStation, PCSX2, Dolphin

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by Gamemulatorer** [comment]
Create game config per game.
![IMG_20221013_220516](https://user-images.githubusercontent.com/37603562/195619104-2ac6af25-80e4-4b86-ba82-fc922c95f9da.jpg)

**Comment by unknownbrackets** [comment]
This is probably essentially a duplicate of #15636.  It's only not per game because it can't be switched without a restart.

-[Unknown]

**Comment by iota97** [comment]
I don't think that this is a duplicate, while it would be better to change BE on the fly, I think this is more about saving the right BE.

It's possible to restart PPSSPP and enqueue the game start for this.

**Comment by kathyrollo** [comment]
Hi, not necessarily a dupe because you can change BE without restart yet it still changes it for global setting rather than per game setting.

Changing BE without restart while keeping it for that game only can co-exist as with other emulators.
